### PR TITLE
Fixed #599 - change Cancel popin button from save to confirm

### LINKED ISSUE #599: In cancel pop in put "confirm" and not "save" in the button
In cancel pop in put "confirm" and not "save" in the button

### PR DESCRIPTION
**Pull request by mickaelandrieu**
A short fix about #599 : only need to change "Save" to "Confirm".

### CODE REVIEW

File: src/tb/i18n/fr_FR/global.json [hunk 1]
```diff
@@ -26,7 +27,7 @@
     "contents": "Contenus",
     "contents_validated": "Contenus validés",
     "contents_canceled": "Contenus annulés",
-    "Copy": "Copier",
```
Comment (fkroockmann) [inline src/tb/i18n/fr_FR/global.json]: Thanks dude

**Comment by fkroockmann** [comment]
:+1

**Comment by mickaelandrieu** [comment]
@ndufreche I don't think we need to wait for travis for this kind of stuff ... can you merge ?

**Comment by ndufreche** [comment]
:+1:

**Pull request merged**